assets/inference/environments/minimal-ubuntu22.04-py311-cpu-inference/spec.yaml

```diff
@@ -0,0 +1,18 @@
+$schema: https://azuremlschemas.azureedge.net/latest/environment.schema.json
+
+description: >-
+  AzureML minimal/Ubuntu 20.04/Python 3.9 cpu environment.
```

Comment:
ubuntu 22?

Reply:
Hi vizhur, 
I made changes in the new commit for the above.